nomenclature/processor/region.py

```diff
@@ -315,57 +316,59 @@ def apply(self, df: IamDataFrame, dsd: DataStructureDefinition) -> IamDataFrame:
 
             # If no mapping is defined the data frame is returned unchanged
             if model not in self.mappings:
-                logging.info(f"No region aggregation mapping found for model {model}")
+                logger.info(f"No region aggregation mapping found for model {model}")
                 processed_dfs.append(model_df)
             # Otherwise we first rename, then aggregate
             else:
                 # before aggregating, check that all regions are valid
                 self.mappings[model].validate_regions(dsd)
-                logging.info(
+                logger.info(
                     f"Applying region aggregation mapping for model {model} from file "
                     f"{self.mappings[model].file}"
                 )
-                # Rename
-                if self.mappings[model].native_regions is not None:
-                    processed_dfs.append(
-                        model_df.filter(
-                            region=self.mappings[model].model_native_region_names
-                        ).rename(region=self.mappings[model].rename_mapping)
-                    )
 
-                # Aggregate
-                if self.mappings[model].common_regions is not None:
-                    vars = self._filter_dict_args(model_df.variable, dsd)
-                    vars_default_args = [
-                        var for var, kwargs in vars.items() if not kwargs
-                    ]
-                    vars_kwargs = {
-                        var: kwargs
-                        for var, kwargs in vars.items()
-                        if var not in vars_default_args
-                    }
-                    for cr in self.mappings[model].common_regions:
-                        regions = [cr.name, cr.constituent_regions]
-                        # First perform 'simple' aggregation (i.e. no aggregation args)
+                with adjust_log_level():  # silence empty filter
```

Comment:
I looked up the default value for the log level which I didn't know was "ERROR". Maybe just to make it clear to the reader you could explicitly set it.
```suggestion
                with adjust_log_level(level="ERROR"):  # silence empty filter
```